src/pages/nft/collection/_collectionId/index.vue

```diff
@@ -55,7 +55,7 @@
       :creator-avatar="creatorAvatar"
       :display-name="creatorDisplayNameFull"
       :currency="defaultCurrency"
-      :class-id="classId"
+      :class-id="collectionId"
```

Comment:
Please add a new prop `collection-id` and handle the logic inside `NFTBookTippingDialog`, using `collectionId` for `class-id` will make later reader very confused

Reply:
[🎨 Add props collectionId to TippingDialog](https://github.com/likecoin/liker-land/pull/1582/commits/cc5080dc37099fac748b24e27420f2f41ccdc324)